src/test/java/com/plaid/client/integration/PaymentCreateTest.java

```diff
@@ -74,4 +94,31 @@ public void testStandingOrderCreateSuccess() throws Exception {
     assertNotNull(response.body().getPaymentId());
     assertNotNull(response.body().getStatus());
   }
+
+  @Test
+  public void testSingleImmediatePaymentWithRequestRefundDetailsCreateSuccess() throws Exception {
+    PaymentOptions options = new PaymentOptions().withRefundDetails(true);
+    Response<PaymentCreateResponse> response = createSingleImmediatePaymentWithOptions(client(), options);
+    assertSuccessResponse(response);
+    assertNotNull(response.body().getPaymentId());
+    assertNotNull(response.body().getStatus());
+  }
+
+  @Test
+  public void testSingleImmediatePaymentWithBacsCreateSuccess() throws Exception {
+    PaymentOptions options = new PaymentOptions().withBacs(new Bacs("1234567890", "000000"));
+    Response<PaymentCreateResponse> response = createSingleImmediatePaymentWithOptions(client(), options);
+    assertSuccessResponse(response);
+    assertNotNull(response.body().getPaymentId());
+    assertNotNull(response.body().getStatus());
+  }
+
+  @Test
+  public void testSingleImmediatePaymentWithIbanCreateSuccess() throws Exception {
+    PaymentOptions options = new PaymentOptions().withIban("iban");
+    Response<PaymentCreateResponse> response = createSingleImmediatePaymentWithOptions(client(), options);
+    assertSuccessResponse(response);
+    assertNotNull(response.body().getPaymentId());
+    assertNotNull(response.body().getStatus());
```

Comment:
Should we be testing that the values are set correctly on the payment? (Currently these fields may not be making it to the server correctly and we'd have no way of knowing)

Reply:
I've updated the new tests to make an additional `get` request to validate the data is returned.